app/templates/index.html

```diff
@@ -1,36 +1,43 @@
-{% extends "base.html" %}
-
+{% extends "base.html" %} 
 {% block head %}
-<link rel="stylesheet" href="/static/css/popup.css">
-{% endblock %}
-
-{% block content %}
+	<link rel="stylesheet" href="/static/css/popup.css" />
+{% endblock %} 
+{% block content %} 
 {# Hack to fix strange onclick bubbling behavior on mobile Safari #}
 <div id="controls" onclick="void(0);">
-    <input type="text" id="query" placeholder="Search">
-    <button id="submit"><i class="fa fa-search"></i></button>
-    <div class="filters">
-        <button id="clear_filters">
-            <i class="fa fa-filter"></i>
-            Clear Filters
-        </button>
-        {% for category in filters %}
-            {% for filter in filters[category] %}
-                {% with name = filter, items = options[filter], header=filters[category][filter]['header'], default = filters[category][filter]['default'] %}
-                    {% include "_filter.html" %}
-                {% endwith %}
-            {% endfor %}
-        {% endfor %}
-    </div>
+	<div id="control_bar">
+		<div class="search_wrapper">
+			<input type="text" id="query" placeholder="Search" />
+			<button id="submit"><i class="fa fa-search"></i></button>
+		</div>
+
+		<div class="filters" id="filters">
+			<button id="clear_filters">
+				<i class="fa fa-filter"></i>
+				Clear Filters
+			</button>
+			{% for category in filters %} 
+				{% for filter in filters[category] %} 
+					{% with name = filter, items = options[filter], header=filters[category][filter]['header'], default = filters[category][filter]['default'] %} 
+						{% include "_filter.html" %} 
+					{% endwith %} 
+				{% endfor %} 
+			{% endfor %}
+			<button id="expand_filters">
+				<i class="fa fa-filter"></i>
```

Comment:
Yeah as stated above can we find a different icon for this one

Reply:
I'd also be okay with hiding the clear filters button in the advanced section, but whatever you prefer is ok

Reply:
sorry missed that one, switched to chevron-circle-down

Reply:
Isn't that what we use for everything else?

Reply:
no, we use chevron-down for everything else